pkg/lib/credentials.js

```diff
@@ -36,7 +36,7 @@ function Keys() {
 
     cockpit.event_target(this);
 
-    cockpit.user()
+    self.p_have_path = cockpit.user()
```

Comment:
"p" for promise?

Reply:
Yes. I can spell it out if you prefer.

Reply:
I'm OK with the `p`, and now that I know what it means, I even prefer it.  I just didn't know the convention.